cli_reference/basic_cli_operations.adoc

```diff
@@ -136,32 +190,70 @@ One of the fundamental capabilities of OpenShift is the ability to build applica
 |Retrieves the build logs for the specified build.
 |===
 
-== Deployment CLI Operations
-OpenShift provides CLI access to inspect and manipulate deployment configurations using standard `osc` resource operations such as `get`, `create`, and `describe`.
-
-=== Deployment Configurations
+== Deployment Operations
+OpenShift provides CLI access to inspect and manipulate
+link:../dev_guide/deployments.html[deployment configurations] using standard
+`osc` resource operations such as `get`, `create`, and `describe`.
 
-The syntax to describe a deployment configuration in human-readable form is:
+To describe a deployment configuration in human-readable form:
 
```

Comment:
The sentence "To describe a deployment configuration in human-readable form:" isn't very clear as to what you're doing. It's better to state that you use this syntax to describe a deployment config.

[Suggestion]
Use the following syntax to describe a deployment configuration in human-readable form: